projects/ondofinance/index.js

```
@@ -84,30 +95,39 @@ async function tvl(timestamp, block, chainBlocks) {
     return balances;
 }
 
-function tvlForAllPair(allPairVault) {
-    return async (timestamp, block) => {
+function tvlForAllPairs(timestamp, block, chainBlocks) {
+    let ondoAllPairVaults = [oldAllPairVault, newAllPairVault]
+    try {
+        const data = await (await fetch("https://data.ondo.finance/v1/addresses")).json()
+        ondoAllPairVaults = data["ondo_all_pair_vaults"]
+    } catch {
+        console.log("Failed to data from Ondo Finance, resorting to defaults")
```

same comment as above, please throw an error here.